README.md

```diff
@@ -153,3 +153,19 @@ import numpy as np
 |           `-- training_samples_compiled/
 `-- tests/
 ```
+
+# References
+
+In case you are using MQT Predictor in your work, we would be thankful if you referred to it by citing the following publication:
+
+```bibtex
+@misc{quetschlich2022mqtpredictor,
+  title={Predicting Good Quantum Circuit Compilation Options},
+  shorttitle = {{{MQT Predictor}}},
```

Comment:
Do we really want to use this as a short title since it is not really related to the overall title? I think I am slightly  in favor of keeping it. Just wanted to bring it up.

Reply:
True, good point. I am with you and do also slightly favor keeping it.